package-lock.json

```diff
@@ -1,7109 +0,0 @@
-{
```

Comment:
Is it expected that this file is deleted?

Reply:
No, I'll check

Reply:
Re added